searchcore/src/vespa/searchcore/proton/documentmetastore/i_document_meta_store.h

```diff
@@ -4,6 +4,7 @@
 
 #include "lid_gid_key_comparator.h"
 #include "i_simple_document_meta_store.h"
+#include "operation_listener.h"
```

Comment:
This include is not needed if documentmetastore::OperationListener is forwared declared later on in this file.

Reply:
Fixed